### PR TITLE
Karizma add links

### PR DESCRIPTION
**Pull request by karizma**
Added 11 sites,

11th commit typo: `added glassdoor` => `added glitch`

### CODE REVIEW

**Comment by jtagcat** [comment]
Please address: `Line: 3396 Unexpected trailing spaces found. [warning]`

**Comment by karizma** [comment]
>Please address: Line: 3396 Unexpected trailing spaces found. [warning]

Wasn't aware of it, I used https://jsonlint.com/ to validate and it didn't give any errors or warnings, will fix.

> Also why would you need to re-add the translation file? Rather use git to undo the changes to the file.

I was planning on that, but by the time I realized it was already the 11th commit, and I thought if I reverted back to the file I would have to redo all the commits. Also since I'm not too experienced with git, I didn't want to cause more issues.

**Comment by jtagcat** [comment]
> > Please address: Line: 3396 Unexpected trailing spaces found. [warning]
> 
> Wasn't aware of it, I used https://jsonlint.com/ to validate and it didn't give any errors or warnings, will fix.
> 
> > Also why would you need to re-add the translation file? Rather use git to undo the changes to the file.
> 
> I was planning on that, but by the time I realized it was already the 11th commit, and I thought if I reverted back to the file I would have to redo all the commits. Also since I'm not too experienced with git, I didn't want to cause more issues.

Ah okay, I troubled with git for like a year.. The golden goal is to have no unnecessary commits, and not touch any unneeded files (so if somebody comes to the file's history, you would show up as the last person who edited it).

We can let this be.

**Comment by jtagcat** [comment]
On GH, you can click details on the tests. Did it for you:
```
Sorting error in _data/sites.json

Keys must be in alphanumeric order. sellfy needs to come after noun project

The command "./script/validate_json.rb" exited with 2.
```

**Comment by karizma** [comment]
Yeah I was just checking the error details

> (T comes after S, but The does not count.)

Looks like you are right, noticed that with `The Artist Union` as well.

Is there any way I could have tested before I pull requested?

**Comment by jtagcat** [comment]
> Yeah I was just checking the error details
> 
> > (T comes after S, but The does not count.)
> 
> Looks like you are right, noticed that with `The Artist Union` as well.
> 
> Is there any way I could have tested before I pull requested?

There are draft pull requests.

In theory, you can copy the tests to your repo somehow, but I have never tried it.

**Comment by karizma** [comment]
Sorry for all the unnecessary commits, tried to revert back to the commit before "fixed position for sellfy" but it ended up with more commits, feel free to close and add the changes yourself if you like.

**Comment by jtagcat** [comment]
One thing I think you are missing in git is force pushing. If you hard reset, then it's 'oh github version is newer, let's download and merge and everything'. In the case when you hard reset, fix your mistakes and commit like nothing happened, you need to push with the -f flag.

File: _data/sites.json [hunk 1]
```diff
@@ -7123,7 +7211,7 @@
             "scentbird.com"
         ]
     },
-
+
```
Comment (jtagcat) [inline _data/sites.json:7214]: This should do it:
```suggestion

```

**Comment by tupaschoal** [comment]
> Wasn't aware of it, I used https://jsonlint.com/ to validate and it didn't give any errors or warnings, will fix.

Our checks are a bit stricter, the trailing spaces themselves won't make the JSON invalid, but we just like to keep it tidy

> Is there any way I could have tested before I pull requested?

Yes, if you clone locally and follow [the instructions we have on our readme](https://github.com/jdm-contrib/jdm#building-locally), but typically you can make a draft PR and fix on the spot, it's alright too.

> Sorry for all the unnecessary commits, tried to revert back to the commit before "fixed position for sellfy" but it ended up with more commits, feel free to close and add the changes yourself if you like.

No problem, in the end we always squash and merge, so it will go away.

File: _data/sites.json [hunk 2]
```diff
@@ -7123,7 +7211,6 @@
             "scentbird.com"
         ]
     },
-
     {
```
Comment (tupaschoal) [inline _data/sites.json:7214]:
```suggestion

    {
```

File: _data/sites.json [hunk 3]
```diff
@@ -1533,6 +1543,15 @@
         ]
     },
 
+    {
+        "name": "Codetasty",
+        "url": "https://codetasty.com/settings/security",
+        "difficulty": "easy",
+        "domains": [
```
Comment (tupaschoal) [inline _data/sites.json:1550]: Can you add notes explaining what to look for?

File: _data/sites.json [hunk 4]
```diff
@@ -5747,6 +5826,15 @@
         ]
     },
 
+    {
+        "name": "Nintendo",
+        "url": "https://accounts.nintendo.com/withdraw/confirm",
+        "difficulty": "easy",
```
Comment (tupaschoal) [inline _data/sites.json:5832]: Can you add notes explaining what to look for?

File: _data/sites.json [hunk 5]
```diff
@@ -7142,6 +7229,15 @@
         ]
     },
 
+    {
+        "name": "Sellfy",
+        "url": "https://sellfy.com/user/settings/account",
+        "difficulty": "easy",
```
Comment (tupaschoal) [inline _data/sites.json:7235]: Can you add notes explaining what to look for?

**Comment by karizma** [comment]
> I've added a few comments requiring notes on some entries that didn't have it, because sometimes though it is easy, just clicking a button, the button might be on a specific tab or menu option and it would be great if we can provide it.

All of the links I did not specify notes for didn't have any tabs or menus, most were just scroll down and press the delete button

Although some of them still could benefit from other notes which I added.

**Comment by tupaschoal** [comment]
It's alright, I think it looks great now, thanks for your contribution!

**Pull request merged**